src/Orchard.Cms.Web/Modules/Orchard.OpenId/Views/Access/Authorize.cshtml

```diff
@@ -0,0 +1,16 @@
+﻿@using Orchard.OpenId.ViewModels
+@model AuthorizeViewModel
+
+<div class="jumbotron">
+    <h1>Authorization</h1>
+    
+    <p class="lead text-left">@T["Do you want to grant <strong>{0}</strong> access to your data? (scopes requested: {1})", @Model.ApplicationName, @Model.Scope]</p>
+
+    <form method="post">
+        @Html.AntiForgeryToken()
+
+        <input type="hidden" name="request_id" value="@Model.RequestId" />
```

Comment:
I've replaced `name="request_id" value="@Model.RequestId"` per `asp-for="RequestId"` but then it fails. In fact, RequestId has annottation `[BindNever]` https://github.com/OrchardCMS/Orchard2/blob/oauth2/src/Orchard.Cms.Web/Modules/Orchard.OpenId/ViewModels/AuthorizeViewModel.cs#L15 so, for some reason it is needed to use name "request_id". So, finally I left `asp-for="RequestId" name="request_id"`
@PinpointTownes Why is this needed?